src/components/CoinListItem/index.js

```diff
@@ -34,53 +46,81 @@ export const CoinListItem = (
   },
   ...rest
 ) => {
+  const currencySymbol = getCurrencySymbol(currency);
   return (
-    <StyledRow>
-      <Favorite span={1}>
-        <CoinFavorite />
-      </Favorite>
-      <Rank span={1}>#{rank}</Rank>
-      <Col span={1}>
-        <Img src={img} />
-      </Col>
-      <Name span={3}>
-        <StyledLink to={`/coin/${name}`}>{name}</StyledLink>
-      </Name>
-      <Col span={1}>
-        <Ticker>{ticker}</Ticker>
+    <ListItemRow>
+      <FavoriteCol lg={{ span: 1 }}>
+        <FavoriteCoin />
+      </FavoriteCol>
+      <RankCol lg={{ span: 1 }}>#{rank}</RankCol>
+      <ImgCol lg={{ span: 1 }}>
+        <img src={img} />
+      </ImgCol>
+      <NameCol lg={{ span: 4 }}>
+        <div>
+          <StyledLink to={`/coin/${id}`}>{name}</StyledLink>
+        </div>
+        <div>
+          <Ticker>{ticker}</Ticker>
+        </div>
+      </NameCol>
+      <CurrentPriceCol lg={{ span: 1 }}>
+        {currencySymbol}
+        {formatPrice(currentPrice)}
+      </CurrentPriceCol>
+      <Col lg={{ span: 5 }}>
+        {/* // Try a map here...............................................................
+        // Use the .value prop to create 1 condition for all 3 colums in the styles.js fiel */}
+        <Row>
```

Comment:
Same thing here, I believe this PriceChange elements could be more dynamic I believe

Reply:
I did the map for this both in the sort header and in the List item price